superset/db_engine_specs/README.md

```diff
@@ -547,65 +547,53 @@ Alternatively, it's also possible to impersonate users by implementing the `upda
 
 Support for authenticating to a database using personal OAuth2 access tokens was introduced in [SIP-85](https://github.com/apache/superset/issues/20300). The Google Sheets DB engine spec is the reference implementation.
 
-To add support for OAuth2 to a DB engine spec, the following attribute and methods are needed:
+To add support for OAuth2 to a DB engine spec, the following attributes are needed:
```

Comment:
As the API is still evolving, I think it could be a god idea to clearly state that this functionality is still experimental, and subject to breaking changes.

Reply:
Ah, good point, I'll a note — though I am hoping we can stabilize the API in the next couple months.